Scripts/Spice.py

```diff
@@ -17,14 +18,188 @@ def compute_view_frustrum(instrument):
 
 def spacecraft_position_and_orientation(spacecraft, point):
     """
-    stub
+    Look under SPK Kernel for now
     :param spacecraft: Data structure that models the spacecraft.
     :param point: point in time.
     :return:
     """
     pass
 
-
 """
 Create Spice Wrappers Here:
-"""
\ No newline at end of file
+"""
+
+###########
+# Kernels #
+###########
+
+class Kernel(str):
+    """
+    Enum types for kernels
+
+    for more info about each type refer to:
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/Tutorials/pdf/individual_docs/12_intro_to_kernels.pdf
+    """
+    spk = 'spk'     #**Spacecraft and Planet Ephemeris
+    pck = 'pck'     #Planetary Constants
+    ik = 'ik'       #**Instrument
+    ck = 'ck'       #**Camera Matrix (Orientation)
+    ek = 'ek'       #Events
+    fk = 'fk'       #**Reference Frame Specification
+    sclk = 'sclk'   #**Spacecraft Clock Correlation
+    lsk = 'lsk'     #**Leap Seconds (Time)
+    mk = 'mk'       #**Meta-Kernel (for loading many kernels)
+    dsk = 'dsk'     #Digital Shape Kernel
+    dbk = 'dbk'     #Database Mechanism
+
+def load_kernel(fileName : str):
+    """
+    Load single or multiple kernels (through meta-kernel file, .mk)
+
+    :param fileName: str file path of kernel
+    """
+    spice.furnsh(fileName)
+    return
+
+def unload_kernel(fileName : str):
+    """
+    Unload a single or multiple kernels (through meta-kernel file, .mk)
+    :param fileName: str file path of kernel
+    """
+    spice.unload(fileName)
+    return
+
+def unload_all_kernels():
+    """
+    Unloads all kernels
+    """
+    spice.kclear()
+    return
```

Comment:
Empty returns aren’t necessary, just omit it.

Reply:
```suggestion

```

Reply:
https://github.com/kylrbutlr/CraterCapstone/blob/6a65b69bac6a288f1cd591719e21da8e22d2fa1f/Scripts/Spice.py#L75